bus-mapping/src/evm/opcodes.rs

```diff
@@ -285,7 +288,11 @@ fn fn_gen_error_state_associated_ops(error: &ExecError) -> Option<FnGenAssociate
         ExecError::InsufficientBalance => Some(CallOpcode::<7>::gen_associated_ops),
         ExecError::WriteProtection => Some(ErrorWriteProtection::gen_associated_ops),
         ExecError::ReturnDataOutOfBounds => Some(ErrorReturnDataOutOfBound::gen_associated_ops),
-
+        ExecError::Depth => {
+            let op = geth_step.op;
+            assert!(op.is_call());
```

Comment:
if !op.is_call() {
    unimplemented!("ErrDepth in create will be implemented later");
}